source/linux/Foundational_Components/Kernel/Kernel_Drivers/Display/DSS7.rst

```diff
@@ -362,7 +362,12 @@ Device Tree Node
 Documentation for tidss device tree node and its properties can be found in linux kernel device tree bindings in below directory
 ``Documentation/devicetree/bindings/display/ti/``. Seperate binding files are present for different version of the DSS controller.
 
+.. ifconfig:: CONFIG_part_variant in ('J721S2')
 
+        - **MCASP1_ACLKX.DP0_HPD** is required for DP. Although it is not connected, it cannot be used as GPIO.
```

Comment:
MCASP1_ACLKX.DP0_HPD does not mention full scope of pin. 
You may use Ball number AA24..Something like 
Ball number AA24 is used for HPD in mux mode 3, irrespective of this is connected as HPD or not, this pin can not be used for other purpose, if DP0 is used.

Reply:
be careful about Ball naming -> you have to add the package type with the ball name